### PR TITLE
[original author: mrnikwaws] Neuron glu operator support

### PR DESCRIPTION
**Pull request by aws-kingrj**
- Adding glu operator support and cpp test
- original author mrnikaws
- tested internally with AWS Neuron

### CODE REVIEW

File: torch_xla/csrc/ops/ops_xla_shape_fn.cpp [hunk 1]
```diff
@@ -421,6 +421,21 @@ xla::Shape GeTensorOutputShape(const torch::lazy::Value& self,
...
+  std::vector<int64_t> output_sizes(std::begin(inp_dimensions),
+                                    std::end(inp_dimensions));
+ 
+  // Output shape is always half the input shape on the specified dimension
+  output_sizes[dim] = inp_dimensions[dim] / 2;
+ 
+  return xla::ShapeUtil::MakeShape(input_shape.element_type(), output_sizes);
+}
+
 xla::Shape GtScalarOutputShape(const torch::lazy::Value& self,
```
Comment (JackCaoG) [inline torch_xla/csrc/ops/ops_xla_shape_fn.cpp:439]: can you add a cpp test similar to https://github.com/pytorch/xla/blob/028df4da388468fa9a41b1f98ea08bfce13b4c63/test/cpp/test_aten_xla_tensor_5.cpp#L209 ?

Comment (aws-kingrj) [reply torch_xla/csrc/ops/ops_xla_shape_fn.cpp:439]: There should be a cpp test already added or another one?

Comment (JackCaoG) [reply torch_xla/csrc/ops/ops_xla_shape_fn.cpp:439]: you should add another one similar to Gelu.

Comment (mrnikwaws) [reply torch_xla/csrc/ops/ops_xla_shape_fn.cpp:439]: Hey Jack can you help me understand what is missing?  

The test in https://github.com/pytorch/xla/pull/5466/files#diff-0c11cc2fbeb0cfbc8fe223f9ac3df0904dcbb1277946a55a56cb102642f4faefR1322-R1342 was intended to be similar to the Gelu one, it tests a variety of input permutations across both input variables and uses the same checks.

Comment (JackCaoG) [reply torch_xla/csrc/ops/ops_xla_shape_fn.cpp:439]: You can build it with `BUILD_CPP_TESTS=1` and run CPP test with `./test/cpp/run_test.sh`

**Comment by JackCaoG** [comment]
@aws-kingrj can you rebase this pr? I should fix the build timeout.

**Pull request closed without merging**